cwl/src/main/scala/cwl/CwlType.scala

```diff
@@ -235,12 +235,8 @@ case class Directory private
 
   lazy val asWomValue: ErrorOr[WomFile] = {
     errorOrListingOption flatMap { listingOption =>
-      val valueOption = path.orElse(location).orElse(basename)
-      (valueOption, listingOption) match {
-        case (None, None) =>
-          "Cannot convert CWL File to WomValue without either a location, a path, a basename, or a listing".invalidNel
-        case (_, _) => WomMaybeListedDirectory(valueOption, listingOption).valid
-      }
+      val valueOption = path.orElse(location).orElse(basename).orElse(Option(better.files.File.newTemporaryDirectory().pathAsString))
```

Comment:
Is this final `Option` intended to always be `Some`? Could the last `orElse` be a `getOrElse`?

Reply:
Not sure if I can de-`Option` the `valueOption`, I'll look at that.

Reply:
I tried this and hit hanging tests in sbt that I couldn't resolve in a timely manner so leaving as is for now.